### ISSUE TITLE
Expose raw values for color properties

### ISSUE THREAD

**Issue by ecstasy2** [tachyons-css/react-native-style-tachyons#45]
Some libraries don't support customization via the `style` property and instead expect a raw color to be passed.
Such a prominent library is `https://github.com/oblador/react-native-vector-icons`. Exposing the raw colors the same way we expose sizes will be very useful when working with such library.

**Comment by ecstasy2** [comment]
I am happy to send a PR if this is something that is of interest.

**Comment by fab1an** [comment]
Have you tried setting the color via `cls`?

**Comment by ecstasy2** [comment]
Yes I tried. It won't work. The component is expecting the color as the
`color` attribute.

**Comment by fab1an** [comment]
Curious. In the documentation it says that `style` works (example: https://github.com/oblador/react-native-vector-icons#styling)

**Comment by ecstasy2** [comment]
@fab1an Thanks for pointing this out. I think I was misusing it somehow. Either way I think having the ability to get the colors and all their variants would be useful. What do you think?

**Comment by fab1an** [comment]
Which version of rnt are you using? Since `3.0` colors aren't generated or modified anymore, they are just passed through to react-native.

**Comment by ecstasy2** [comment]
I am using the latest version. I think not modifying the colors make sense given that it is so easy to manually add color variants.

What I was interested in is having access to the colors the same way we have for sizes:

```javascript
import NativeTachyons, { colors , sizes } from 'react-native-style-tachyons';
```

Looking at old version of the library, it seems like that was the case before. Any reason it was removed?
